src/components/courses/course.constant.js

```diff
@@ -0,0 +1,34 @@
+export const coursesColumnConfig = [
```

Comment:
This should be all caps with underscores because this is a static constant array. Might as well wrap it in `Object.freeze` to make it read only as well.

```js
export const COURSES_COLUMN_CONFIG = Object.freeze([
  { ... },
]);
```